sage-ios/SAGE/SAGE/AnnouncementsDetailView.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 class AnnouncementsDetailView: UIView {
     var scrollView = UIScrollView()
     let announcementUserImg = ProfileImageView()
+    let announcementUserButton = UIButton()
```

Comment:
noooooooo dont  do this. you dont need a button, you can add a TapGestureRecognizer on the announcementUserImg instead